lib/tasks/jenkins.rake

```diff
@@ -0,0 +1,23 @@
+begin
+  require 'English'
```

Comment:
`English` is part of Ruby itself, so I don't think you need to rescue it. If that's broken, something is seriously wrong.

Having said that: if you change the rubocop task (or drop it altogether) to not use `system()` you don't need `$CHILD_STATUS` and can safely drop it.